tests/models/test_model_input_examples.py

```diff
@@ -114,6 +122,17 @@ def test_input_examples(pandas_df_with_all_types, dict_of_ndarrays):
             .all()
         )
 
+    with TempDir() as tmp:
```

Comment:
could this be split out into its own test?

Reply:
done!